UDF are not working anymore after upgrade of Exasol 6.2.0 to 7.0.3

Hello,

after upgrading Exasol from 6.2.0 to 7.0.3 the Hadoop-UDF aren't working anymore.
Here the error message:

```
SQL Error [22002]: VM error: F-UDF-CL-LIB-1127: F-UDF-CL-SL-JAVA-1002: F-UDF-CL-SL-JAVA-1013: 
com.exasol.ExaUDFException: F-UDF-CL-SL-JAVA-1080: Exception during run 
java.lang.ExceptionInInitializerError
org.apache.hadoop.util.ReflectionUtils.<clinit>(ReflectionUtils.java:137)
org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.initRegistry(MetricsSourceBuilder.java:92)
org.apache.hadoop.metrics2.lib.MetricsSourceBuilder.<init>(MetricsSourceBuilder.java:56)
org.apache.hadoop.metrics2.lib.MetricsAnnotations.newSourceBuilder(MetricsAnnotations.java:42)
org.apache.hadoop.metrics2.impl.MetricsSystemImpl.register(MetricsSystemImpl.java:223)
org.apache.hadoop.metrics2.MetricsSystem.register(MetricsSystem.java:60)
org.apache.hadoop.security.UserGroupInformation$UgiMetrics.create(UserGroupInformation.java:120)
org.apache.hadoop.security.UserGroupInformation.<clinit>(UserGroupInformation.java:236)
com.exasol.hadoop.HdfsSerDeImportService.importFiles(HdfsSerDeImportService.java:88)
com.exasol.hadoop.scriptclasses.ImportHiveTableFiles.run(ImportHiveTableFiles.java:84)
com.exasol.ExaWrapper.run(ExaWrapper.java:196)
Caused by: java.lang.NullPointerException
java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2646)
java.base/java.lang.Runtime.loadLibrary0(Runtime.java:830)
java.base/java.lang.System.loadLibrary(System.java:1873)
java.management/java.lang.management.ManagementFactory.lambda$static$8(ManagementFactory.java:1020)
java.base/java.security.AccessController.doPrivileged(Native Method)
java.management/java.lang.management.ManagementFactory.<clinit>(ManagementFactory.java:1019)
	... 15 more
 (Session: 1684314033601904644)
```

Regards,

Nicola

--- a/doc/changes/changes_1.0.0.md
+++ b/doc/changes/changes_1.0.0.md
@@ -3,3 +3,4 @@
 ## Bugfixes
 
 * #56: Fixed Hive 3.0.0+ version Metastore URI selection issue (PR #57)
+* #52: Fixed native library loading issue in 6.2.12+ and 7.0.0+ versions (PR #58)

--- a/hadoop-etl-common/src/main/java/com/exasol/hadoop/NativeHadoopLibUtils.java
+++ b/hadoop-etl-common/src/main/java/com/exasol/hadoop/NativeHadoopLibUtils.java
@@ -7,8 +7,8 @@ import java.lang.reflect.Field;
 import java.util.Map;
 
 public class NativeHadoopLibUtils {
-    
-    public static void initHadoopNativeLib() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
+
+    public static void initHadoopNativeLib() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException, IOException {
         // This path must exist at program startup and must be in LD_LIBRARY_PATH. Otherwise the dynamic linker seems to ignore it
         final String nativeHadoopLibDir = "/tmp/";
         NativeHadoopLibUtils.writeNativeHadoopLibs(nativeHadoopLibDir);
@@ -36,7 +36,7 @@ public class NativeHadoopLibUtils {
             e.printStackTrace();
         }
     }
-    
+
     private static void writeNativeHadoopLib(String resourceName, String targetFileName, String targetPath) throws Exception {
         File dirFile = new File(targetPath + targetFileName);
         if (dirFile.exists()) {
@@ -65,23 +65,9 @@ public class NativeHadoopLibUtils {
     }
 
     /**
-     * This enables the java.library.path to be modified at runtime, which is required to load java native libraries
-     * From a Sun engineer at http://forums.sun.com/thread.jspa?threadID=707176
-     * See http://stackoverflow.com/questions/5419039/is-djava-library-path-equivalent-to-system-setpropertyjava-library-path
-     * and http://fahdshariff.blogspot.jp/2011/08/changing-java-library-path-at-runtime.html
+     * This enables the java.library.path to be modified at runtime, which is required to load java native libraries.
      */
-    private static void addDirToJavaLibraryPath(String path) throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
-        System.setProperty("java.library.path", path );
-        Field fieldSysPath = ClassLoader.class.getDeclaredField( "sys_paths" );
-        fieldSysPath.setAccessible( true );
-        fieldSysPath.set( null, null );
-    }
-    
-    /**
-     * This is another variant of addDirToJavaLibraryPath(). Currently unused, because the other should work.
-     */
-    @SuppressWarnings("unused")
-    private static void addDirToJavaLibraryPath2(String s) throws IOException {
+    private static void addDirToJavaLibraryPath(String s) throws IOException {
         try {
             Field field = ClassLoader.class.getDeclaredField("usr_paths");
             field.setAccessible(true);

